Intersect.Server/Networking/PacketSender.cs

```diff
@@ -1003,58 +1003,61 @@ public static void SendCreateCharacter(Client client)
         public static void SendPlayerCharacters(Client client)
         {
             var characters = new List<CharacterPacket>();
-            foreach (var character in client.Characters.OrderByDescending(p => p.LastOnline))
+            if (client.User != null && client.Characters.Count > 0)
```

Comment:
For readability sake, use
```csharp
if (client.User == null || client.Characters.Count < 1)
```
instead of indenting everything it wrap it in an if statement with no else.

Reply:
I think the commit below is what you're looking for but let me know.